### PR TITLE
Iss1956 - Add floating IP pool to request to create floating IP

### PR DESCRIPTION
**Pull request by jadecarino**
## Why?

- To fix defect [OpenStack create floating IP Bad Request](https://github.com/galasa-dev/projectmanagement/issues/1956). Now that there is more than one external network available in OpenStack (created due to requirements for an external ipv6 network), the request to create a floating IP must include which floating IP pool to create it in. I have adapted the Openstack Manager to do this and created a CPS property for the floating IP pool value so it's configurable.
- Also corrected comments and exceptions in the Openstack manager that did not match what they were doing. 
- Removed remaining `@author` tags in this bundle as upgrading it anyway.
- Upgraded version in release.yaml and build.gradle.

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ */
+package dev.galasa.openstack.manager.internal.properties;
+
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.cps.CpsProperties;
+import dev.galasa.openstack.manager.OpenstackManagerException;
+
+/**
+ * OpenStack Floating IP Pool
+ * <p>
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java:14]: Don't do this `<p>` stuff. Just leave a blank line. 
It has the same effect, and it's easier to read the code.

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * The Openstack Floating IP Pool that the OpenStack Manager will use
+ * to create a Floating IP address within.
+ * </p>
+ * <p>
+ * The property is:-<br>
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java:19]:
```
 * The property is :-
 * <code>openstack.server.floatingip.pool=my_network_name</code>
 *```
 That's a much better way of doing javadoc.

**Review by techcobweb — APPROVED** [review]

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java (hunk 1 above)
Comment (jadecarino) [reply galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java:14]: Okay sure - I had replicated the same format as the other Javadoc comments in this bundle here - will fix in a separate PR as I'd like this to build overnight and run in tomorrow's regression run.

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java (hunk 2 above)
Comment (jadecarino) [reply galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java:19]: Okay sure - I had replicated the same format as the other Javadoc comments in this bundle here - will fix in a separate PR as I'd like this to build overnight and run in tomorrow's regression run.

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**